holopy/inference/model.py

```diff
@@ -66,11 +67,11 @@ def make_xarray(dim_name, keys, values):
         return xr.DataArray(np.array(values), coords=[keys], dims=dim_name)
 
 
-def make_complex(real, imag):
```

Comment:
What happened to `make_complex`? Is the idea that now we say `n=prior.Gaussian(1.59, 0.1) + 1j * prior.Uniform(0, 1e-3)`?

Reply:
(I'm mostly asking about documentation.)

Reply:
`make_complex` was just a dispatch function used by a Model to unpack a map (e.g. into a scatterer). It became redundant when `ComplexPrior` was rolled into `TransformedPrior` which has its own similar function.